public/js/main.js

```diff
@@ -392,13 +392,37 @@ function updateAlerts() {
   }
 }
 
+function updateSortie() {
+  var sortie = worldState.sortie;
+  
+  if (sortie.length !== 0) {
+	$('#sortietitle').hide();
+	
+	$('#sortieBoss').html(sortie.boss);
+	$('#sortieFaction').html(sortie.faction);
+	$('#sortieList').find('.variant').remove();
+	
+	sortie.variants.forEach(function(variant, index) {
+	  if ($('#variant_' + index).length === 0) {
+		let sortieRow = '<li class="list-group-item list-group-item-borderless variant" id="variant_' + index + '">';
+		
+		sortieRow += '<b>' + variant.missionType + '</b> - ' + '<span>' + variant.modifier + '</span>';
+		sortieRow += '<br /><b>' + variant.node + '</b></li>';
+		
+		$('#sortiebody').before(sortieRow);
+	  }
+	});
+  }
```

Comment:
If sortie is length 0, (during sortie reset time), there's a period of time where this will display old or no data at all.

Possibly unhide sortietitle and put something useful?